WikiDrips/WikiRequest.swift

```diff
@@ -7,17 +7,67 @@
 //
 
 import Foundation
-import Accounts
-import Social
-import CoreLocation
 
 public class WikiRequest
 {
-    public func fetchWikiDoc(handler: @escaping ([WikiDoc]) -> Void) {
-//      TODO: make api requests
-            var wikiDocs = [WikiDoc]()
-            var wikiDocArray: NSArray?
-        
-
+    let session = URLSession.shared
+    let baseEndpoint: String = "http://en.wikipedia.org/w/api.php?action=query&list=search&srwhat=text&sroffset=0&format=json"
```

Comment:
Consider using `URLComponents` instead of `String`. It has built in checking to make sure your URLs are properly formed.